bigtable-hbase-parent/bigtable-hbase-mapreduce/src/main/java/com/google/cloud/bigtable/mapreduce/ShuffledSequenceFileInputFormat.java

```diff
@@ -0,0 +1,23 @@
+package com.google.cloud.bigtable.mapreduce;
```

Comment:
FYI, you can use [`autogen`](https://github.com/mbrukman/autogen) to add copyright headers to files, and yes, it's a shameless plug. :-)

It automatically uses the current year, of course.